docs/blog/Webots-2023-a-release.md

```diff
@@ -0,0 +1,99 @@
+# Version R2023a Released
+
+<p id="publish-data">By Yannick Goumaz - XXth November 2022</p>
```

Comment:
The date is still missing, is this wanted?

Reply:
Yes, it will be replaced later, once we know it.